code/src/components/ListItem.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { formatDistanceToNow } from 'date-fns';
+
+const ListItem = ({ thoughtId, message, likes, creationDate, handleLikes }) => {
+  const creationDateObject = new Date(creationDate);
+  const formattedCreationDate = formatDistanceToNow(creationDateObject, { addSuffix: true });
+
+  // Two different heart backgrounds depending on if there are likes or not
```

Comment:
This is very well done!

Reply:
Thank you! I stole it from someone else 😅